src/Habits/index.js

```diff
@@ -77,16 +106,16 @@ const Habits = ({ allHabits, selectedDate, onUpdate }) => {
     }
 
     return (
-      <div>
+      <div onContextMenu={(e) => {disableRightclickOnHabitList(e)}}> 
         {habits.map((habit, index) => {
           return (
-            <ClickNHold
-              key={habit.id}
-              time={1}
-              onClickNHold={() => {
-                onClickNHold(habit)
-              }}
-            >
+            <div 
+              onTouchStart={() => handleButtonPress(habit)} 
```

Comment:
Instead of adding so many event listeners, can we have onScroll event with a debounce or throttle (Which ever make sense here) and add a css property called `pointer-event: none` to parent of habits.

By doing this, we can avoid click&hold event and when scroll is done, make pointer-event to initial or remove it.

This is simple and effective fix rather than multiple event listeners